tailwind.config.mjs

```diff
@@ -10,6 +10,9 @@ export default {
   ],
   theme: {
     extend: {
+      borderColor: {
+        DEFAULT: '#dbdbdb',
+      },
```

Comment:
This is available in the next release of the `frontend-shared` package's upstream preset, but not yet...